.github/workflows/dist.yml

```diff
@@ -36,8 +36,8 @@ jobs:
 
       - name: commit
         run: |-
-          git config --global user.name "team-integrations-fnm-bot"
-          git config user.email 'integrationsfnmbot@octopus.com'
+          git config --global user.name "team-github-bot"
+          git config user.email 'team-github-bot@octopus.com'
```

Comment:
I'm assuming that changing this to something like `team-devex-bot` and `team-devex-bot@octopus.com` is not simple?